apis/visibility/v1alpha1/types.go

```diff
@@ -50,7 +50,7 @@ type ClusterQueue struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Summary PendingWorkloadsSummary `json:"pendingworkloadsummary"`
+	Summary PendingWorkloadsSummary `json:"pendingWorkloadsSummary"`
```

Comment:
can you open a separate PR for this?

Reply:
https://github.com/kubernetes-sigs/kueue/pull/1371